components/collapse/style/index.less

```diff
@@ -35,7 +35,7 @@
       position: relative;
       transition: all 0.3s;
 
-      .arrow {
+      .@{collapse-prefix-cls}-arrow {
```

Comment:
This file should not be changed. Because components' style files are synced from antd. We don't modify it.

Reply:
done